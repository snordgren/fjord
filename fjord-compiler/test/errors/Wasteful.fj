module errors.Wasteful

wasteful : Int -* Int -* Int
wasteful a b =
  a
