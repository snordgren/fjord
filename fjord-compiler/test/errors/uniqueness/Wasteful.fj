module errors.uniqueness.Wasteful

wasteful : Int -* Int -* Int
wasteful a b =
  a
