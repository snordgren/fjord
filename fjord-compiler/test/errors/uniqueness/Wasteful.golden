test/errors/uniqueness/Wasteful.fj:5:3:
  |
5 |   a
  |   ^^
too few usages of b
